mkdocs.yml

```
@@ -96,6 +95,7 @@ nav:
         - geps/gep-1686.md
         - geps/gep-1426.md
       - Experimental:
+        - geps/gep-1651.md
         - geps/gep-1748.md
```

Nit: I think the idea is to list these in descending order by GEP number
```suggestion
        - geps/gep-1748.md
        - geps/gep-1651.md
```